src/components/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,9 @@
+import StoreOverview from "@/components/dashboard/store-overview/StoreOverview";
+
+export default function DashboardPage() {
```

Comment:
```suggestion
export default function DashboardPage() {
```
Change the name of the component to StoreOverViewComponent